### ISSUE TITLE
Translation - German

### ISSUE THREAD

**Issue by Omnibrain007** [WaldiPL/webpageScanner#68]
Just done a German translation of the language file. See attached "messages_de.json"
[messages_de.zip](https://github.com/WaldiPL/webpageScanner/files/6249214/messages_de.zip)

**Comment by WaldiPL** [comment]
Hi.
Thanks for this translation. I made a new version, is the following translation correct? 

```
"dropHere":{
    "message": "Drop here to add to the scan list"
  },
"h3sort": {
    "message": "Sorting"
  },
"sort": {
    "message": "Sort"
  },
"descDate": {
    "message": "By newest"
  },
"ascDate": {
    "message": "By oldest"
  },
"az": {
    "message": "From A to Z"
  },
"za": {
    "message": "From Z to A"
  }
```

```
"dropHere":{
    "message": "Hier ablegen, um sie zur Scanliste hinzuzufügen"
  },
"h3sort": {
    "message": "Ordnen"
  },
"sort": {
    "message": "Sortieren"
  },
"descDate": {
    "message": "Vom neuesten"
  },
"ascDate": {
    "message": "Vom ältesten"
  },
"az": {
    "message": "Von A bis Z"
  },
"za": {
    "message": "Von Z bis A"
  }
```

**Comment by Omnibrain007** [comment]
"dropHere":{
    "message": "Hier ablegen, um sie zur Scanliste hinzuzufügen"
  },
"h3sort": {
    "message": "Ordnen"
  },
"sort": {
    "message": "Sortieren"
  },
"descDate": {
    "message": "Nach neuesten"
  },
"ascDate": {
    "message": "Nach ältesten"
  },
"az": {
    "message": "Von A bis Z"
  },
"za": {
    "message": "Von Z bis A"
  }

**Comment by WaldiPL** [comment]
The German language is available in version 2.1.0.
